src/LocalEngine.py

```diff
@@ -860,9 +874,13 @@ def _do_analyze_with_diagnostic(SEED, X_L, X_D, M_c, T, kernel_list, n_steps, c,
                             CT_KERNEL=CT_KERNEL,
                             )
     with gu.Timer('all transitions', verbose=False) as timer:
+        # XXX If doing diagnostics (i.e. invoking from bayeslite) skip the
+        # progress bar.
+        if len(child_n_steps_list) != 1:
```

Comment:
Note that the current timed analysis is actually broken in `master` when using diagnostics, which is the purpose of pull request https://github.com/probcomp/crosscat/pull/108